cdc/cdc.cc

```diff
@@ -194,4 +195,194 @@ future<> setup(service::storage_proxy& proxy, schema_ptr s) {
     });
 }
 
+class transformer final {
+public:
+    struct map_cmp {
+        bool operator()(const std::pair<net::inet_address, unsigned int>& a,
+                        const std::pair<net::inet_address, unsigned int>& b) const {
+            if (a.first.is_ipv4()) {
+                if (b.first.is_ipv4()) {
+                    if (a.first.as_ipv4_address().ip > b.first.as_ipv4_address().ip) {
+                        return false;
+                    }
+                    return a.second < b.second;
```

Comment:
Wasn't that intended to be a lexicographical compare?
In that case, if `a.first.as_ipv4_address().ip < b.first.as_ipv4_address().ip` and `a.second == b.second`, shouldn't it return true?

Reply:
Good catch - will fix. Thanks.